examples/gallery/symbols/basic_symbols.py

```diff
@@ -0,0 +1,116 @@
+"""
+Basic geometric symbols
+-----------------------
+
+The :meth:`pygmt.Figure.plot` method can plot individual geometric symbols
+by passing the corresponding shortcuts to the ``style`` parameter. 
+"""
+
+import pygmt
+
+fig = pygmt.Figure()
+fig.basemap(region=[0, 8, 0, 3], projection="x3c", frame=True)
```

Comment:
This gives a large figure of 24 cm x 9 cm. It would be better to plot a small figure using uppercase **X**, rather than lowercase **x**.
```
fig.basemap(region=[0, 8, 0, 3], projection="x3c", frame=True)
```

Reply:
Changed it, thanks @seisman.